config/FCCAnalysisRun.py

```diff
@@ -1087,7 +1254,10 @@ def run(mainparser, subparser=None):
     if hasattr(args, 'command'):
         if args.command == "run":
             try:
-                runStages(args, rdfModule, args.preprocess, analysisFile)
+                if hasattr(rdfModule, "build_graph"):
```

Comment:
Can you check also if the provided analysis file does not contain `RDFanalysis` class?